### PR TITLE
T20343

### PR DESCRIPTION
**Pull request by tchx84**
https://phabricator.endlessm.com/T20343

### CODE REVIEW

File: ekncontent/ekncontent/eknc-domain.c [hunk 1]
```diff
@@ -457,6 +457,13 @@ eknc_domain_initable_init (GInitable *initable,
       return FALSE;
     }
 
+  if (g_slist_length(self->shards) == 0)
```
Comment (xjuan) [inline ekncontent/ekncontent/eknc-domain.c]: It would be faster to check if its NULL

File: js/app/utils.js [hunk 2]
```diff
@@ -584,3 +584,11 @@ function parse_uri (uri_str, parse_query) {
 
     return {scheme, user, password, host, port, path, query, fragment};
 }
+
+function open_app_center_details (app_id) {
```
Comment (xjuan) [inline js/app/utils.js]: Do you think this will be used in other place?

Comment (tchx84) [reply js/app/utils.js]: nope, as we discussed in slack, I will move it back to the Dialog.

**Comment by tchx84** [comment]
@xjuan thanks, should be ready now.

**Comment by xjuan** [comment]
CR Pass

**Pull request merged**